target/linux/generic/hack-5.4/902-debloat_proc.patch

```diff
@@ -45,7 +45,7 @@ Signed-off-by: Felix Fietkau <nbd@nbd.name>
  	  this interface. For example, rkt is such a piece of software.
  
 +config PROC_STRIPPED
-+	default n
```

Comment:
This is wrong, take a look for example here:

https://github.com/openwrt/openwrt/blob/866307285445f1086af1cc8352a46e55260274d1/config/Config-kernel.in#L34-L36

Reply:
I deleted now the `depends on EXPERT` and moved the config part to `config/Config-kernel.in`

Reply:
Something is wrong.

Reply:
Here is the SWAP config option in kernel. It also has a default value?
https://github.com/torvalds/linux/blob/v5.4/init/Kconfig#L308-L316

So I leaf the proc-patch as it is and I only remove that from config-5.4 and expose it to openwrt buildchain via config/Config-kernel.in?

I think it is now how it should be.